packages/web/src/components/usdc-purchase-details-modal/components/PurchaseModalContent.tsx

```diff
@@ -50,7 +61,67 @@ export const PurchaseModalContent = ({
     }
   }, [track, onClose, goToRoute])
 
-  return (
+  return isPremiumAlbumsEnabled ? (
+    <>
+      <ModalHeader>
+        <ModalTitle title={messages.purchaseDetails} />
+      </ModalHeader>
+      <ModalContent className={styles.content}>
+        <Flex gap='xl' direction='column' w='100%'>
+          <Flex
+            borderBottom='default'
+            gap='l'
+            justifyContent='spaceBetween'
+            w='100%'
+            pb='xl'
+          >
+            <DynamicTrackArtwork
+              id={purchaseDetails.contentId}
+              size={DynamicTrackArtworkSize.LARGE}
+            />
+            <DetailSection label={messages.track}>
+              <TrackLink onClick={onClose} id={purchaseDetails.contentId} />
```

Comment:
we should refactor tracklink to use TextLink so we are up to spec with designs. lmk if you need help crafting that